sources/vcdMaker/src/vcdMaker.cpp

```diff
@@ -45,15 +46,36 @@ int main(int argc, const char *argv[])
     // Source registry.
     SIGNAL::SourceRegistry registry;
 
+    // Line counter.
+    INSTRUMENT::LineCounter *lineCounter = NULL;
```

Comment:
Use `std::unique_ptr` and `std::make_unique` if you must use pointer. I don't like this.